_security/access-control/users-roles.md

```diff
@@ -100,7 +100,7 @@ See [YAML files]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#roles
 
 ### REST API
 
-See [Create role]({{site.url}}{{site.baseurl}}/security/access-control/api/#create-role).
+See [Patch role]({{site.url}}{{site.baseurl}}/security/access-control/api/#patch-role).
```

Comment:
Should this be ({{site.url}}{{site.baseurl}}/security/access-control/api/#patch-role-mapping)?

Reply:
That would be for editing the roles's mappings, not the role itself. In other words, the role needs to be created or edited first before a role can be mapped to a user.

Reply:
We also start talking about this on the line after this section, 105.